specification/operationalinsights/resource-manager/readme.md

```diff
@@ -170,12 +168,31 @@ regenerate-manager: true
 generate-interface: true
 ```
 
-
 ## Suppression
 
 ``` yaml
 directive:
   - from: OperationalInsights.json
-    suppress: R3006  # BodyTopLevelProperties/R3006/RPCViolation
+    suppress: R3006
     reason: properties etag defined as eTag in model
+  - suppress: EnumInsteadOfBoolean
```

Comment:
I believe all of what you're suppressing new here are warnings, is there a reason you'd want to suppress them? There's no need to suppress warnings.

Reply:
Hi Veronica :)
Yes, I'm aware of that. Just trying to be proactive, because they are false positives or by design.

Reply:
Just trying to clear up some of the validation warnings (which is not mandatory)

Reply:
yes, your'e correct. I revered redundant suppression. Thanks